src/snowflake/snowpark/modin/plugin/_internal/join_utils.py

```diff
@@ -1169,7 +1169,7 @@ def convert_index_type_to_variant(
     frame_to_variant = {}
     for frame_id in frame_ids:
         frame_type = frame_id_to_type_map[frame_id]
-        if frame_type not in [VariantType, TimestampType]:
+        if frame_type != VariantType:
```

Comment:
Could be wrong, but shouldn't this be since VariantType is a class:
```suggestion
        if not isinstance(frame_type, VariantType):
```

Reply:
That's right, that should be either VariantType() or use `isinstance`, thanks for catching that!